Fixes some problems with the dark theme

dowdybrown opened the pull request:
This change renders toolbar arrows, separators, and check marks with the correct theme colors. Also added a few menu items to the test program to help verify colors.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/dockpanelsuite/dockpanelsuite?pullRequest=605) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/dockpanelsuite/dockpanelsuite?pullRequest=605) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/dockpanelsuite/dockpanelsuite?pullRequest=605) it.</sub>

dowdybrown commented:
I changed DockSample/MainForm.Designer.cs to add a sub-menu, in order to verify that arrows were drawn in the appropriate color. I also added a disabled item, again to verify the correct colors were being drawn in that situation.

lextm commented:
Thanks. Then we can merge it.
